### ISSUE TITLE
Add Webfont export

### ISSUE THREAD

**Issue by sgigou** [sgigou/NF-Pixels#5]
Also export as WOFF, WOFF2 and EOT.

**Comment by jansteffen** [comment]
Sorry if I'm necroing an abandoned project, but a woff2 export would be lovely. This is the only pixel font I've been able to find that isn't monospaced and is licensed under SIL OFL.

**Comment by sgigou** [comment]
Yeah sadly I am _very_ not working on this… Maybe you can try an online converter? The font don't have any special feature, so it may work properly.
